Dockerfile.cockpit

```diff
@@ -5,5 +5,9 @@ RUN dnf install -y cockpit-ws cockpit-kubernetes && dnf clean all
 RUN echo $'[Negotiate]\nCommand = /usr/libexec/cockpit-stub\n[WebService]\nShell = /shell/simple.html' > /etc/cockpit/cockpit.conf
 RUN rm -rf /usr/share/cockpit/kubernetes/
 
+COPY welder-web*.rpm /tmp/
+RUN dnf -y install /tmp/welder-web*.rpm
+RUN rm -f /tmp/welder-web*.rpm
```

Comment:
I suggest to put that into one `RUN` command (`dnf && rm`), otherwise the `rm` is pointless. Remember that each RUN creates a new docker layer, so effectively you can't remove files in separate steps if you actually want to avoid wasting space.